race/Blue4Eternety; Fallen - A True Awakened Undead Race .json

```diff
@@ -0,0 +1,462 @@
+{
+	"_meta": {
+		"sources": [
+			{
+				"json": "FTAUR:Fallen",

```

Comment:
afraid `:` isn't allowed in `json`/`source`s; please replace with e.g. `FTAURFallen` throughout